packages/internal/src/gql.ts

```diff
@@ -74,3 +78,30 @@ const getFields = (field: FieldNode): any => {
     return obj
   }
 }
+
+export const listQueryTypeFieldsInProject = async () => {
+  try {
+    const schema = await loadSchema(
+      ['graphql/**/*.sdl.{js,ts}', 'directives/**/*.{js,ts}'],
+      {
+        loaders: [
+          new CodeFileLoader({
+            noRequire: true,
```

Comment:
It just parses files, rather than use require. Its the same setting we use in "validateSchema".

I believe we did this because require-ing the files can be slow(er) and also error prone.